src/lib/AppRegistry.tsx

```diff
@@ -109,4 +130,5 @@ AppRegistry.registerComponent("Inbox", () => Inbox)
 AppRegistry.registerComponent("Conversation", () => Conversation)
 AppRegistry.registerComponent("Inquiry", () => Inquiry)
 AppRegistry.registerComponent("Favorites", () => FavoritesScene)
-AppRegistry.registerComponent("BidFlow", () => BidFlow)
+// TODO: Change everything to BidderFlow? AuctionAction?
+AppRegistry.registerComponent("BidderFlow", () => BidderFlow)
```

Comment:
I think I should probably leave this named BidFlow and update it at the same time as `ARBidFlowViewController`'s name and use of this component.